### PR TITLE
[fix] Added priority class to be used in CSI statefulset and daemonset

### PR DESCRIPTION
**Pull request by Ab-hishek**
This PR fixes the issue which is caused during volume creation using cStor pools.

Issue is described below:

### PVC Info:
```
root@ranjith-vm:~# kubectl get pvc
NAME    STATUS    VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS    AGE
minio   Pending                                      cstorsc-oomqo   84s
root@ranjith-vm:~#
root@ranjith-vm:~# kubectl describe pvc minio
Name:          minio
Namespace:     default
StorageClass:  cstorsc-oomqo
Status:        Pending
Volume:
Labels:        app=minio
               chart=minio-7.0.1
               heritage=Helm
               release=minio
Annotations:   volume.beta.kubernetes.io/storage-provisioner: cstor.csi.openebs.io
Finalizers:    [kubernetes.io/pvc-protection]
Capacity:
Access Modes:
VolumeMode:    Filesystem
Mounted By:    minio-b494f5c6-bdz4d
Events:
  Type    Reason                Age               From                         Message
  ----    ------                ----              ----                         -------
  Normal  ExternalProvisioning  2s (x8 over 94s)  persistentvolume-controller  waiting for a volume to be created, either by external provisioner "cstor.csi.openebs.io" or manually created by system administrator
```

### Pool Info:
```
root@ranjith-vm:~# kubectl get cspi -n openebs
NAME                   HOSTNAME                                              FREE     CAPACITY    READONLY   PROVISIONEDREPLICAS   HEALTHYREPLICAS   STATUS   AGE
cstorpool-gawnr-5fvz   gke-stackgres-postgres-m-default-pool-e38f7409-41gf   48200M   48200112k   false      0                     0                 ONLINE   43m
cstorpool-gawnr-6wx8   gke-stackgres-postgres-m-default-pool-e38f7409-9h9b   48200M   48200110k   false      0                     0                 ONLINE   43m
cstorpool-gawnr-7tzq   gke-stackgres-postgres-m-default-pool-e38f7409-pbd5   48200M   48201390k   false      0                     0                 ONLINE   43m
root@ranjith-vm:~# kubectl get cspc -n openebs
NAME              HEALTHYINSTANCES   PROVISIONEDINSTANCES   DESIREDINSTANCES   AGE
cstorpool-gawnr   3                  3                      3                  43m
```

### Deployments & ds in openebs ns:
```
root@ranjith-vm:~# kubectl get deploy,ds -n openebs
NAME                                                             READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/cspc-operator                                    1/1     1            1           3h33m
deployment.apps/cstorpool-gawnr-5fvz                             1/1     1            1           70m
deployment.apps/cstorpool-gawnr-6wx8                             1/1     1            1           70m
deployment.apps/cstorpool-gawnr-7tzq                             1/1     1            1           70m
deployment.apps/cvc-operator                                     1/1     1            1           3h33m
deployment.apps/maya-apiserver                                   1/1     1            1           3h33m
deployment.apps/openebs-admission-server                         1/1     1            1           3h33m
deployment.apps/openebs-cstor-admission-server                   1/1     1            1           3h33m
deployment.apps/openebs-localpv-provisioner                      1/1     1            1           3h33m
deployment.apps/openebs-ndm-operator                             1/1     1            1           3h33m
deployment.apps/openebs-provisioner                              1/1     1            1           3h33m
deployment.apps/openebs-snapshot-operator                        1/1     1            1           3h33m
deployment.apps/pvc-aa2f64ee-3328-4735-a61c-db6f8905535e-ctrl    1/1     1            1           11m
deployment.apps/pvc-aa2f64ee-3328-4735-a61c-db6f8905535e-rep-1   1/1     1            1           11m
deployment.apps/pvc-aa2f64ee-3328-4735-a61c-db6f8905535e-rep-2   1/1     1            1           11m
deployment.apps/pvc-aa2f64ee-3328-4735-a61c-db6f8905535e-rep-3   1/1     1            1           11m
NAME                                    DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR                 AGE
daemonset.apps/openebs-cstor-csi-node   0         0         0       0            0           <none>                        3h33m
daemonset.apps/openebs-ndm              3         3         3       3            3           mayadata.io/data-plane=true   3h33m
```

### Output for describe of csi daeomonset:
```
  HostPathType:       Directory
  Priority Class Name:  system-node-critical
Events:
  Type     Reason        Age                   From                  Message
  ----     ------        ----                  ----                  -------
  Warning  FailedCreate  14m (x30 over 3h34m)  daemonset-controller  Error creating: insufficient quota to match these scopes: [{PriorityClass In [system-node-critical system-cluster-critical]}]
```

So the issue was that the priority class did not get created which is required by the CSI statefulset and ds. This PR adds the priority classes needed by them.

### CODE REVIEW

File: templates/openebs-operator-2.6.0.yaml [hunk 1]
```diff
@@ -5854,4 +5854,20 @@ spec:
...
+  name: system-cluster-critical
+value: 900000000
+---
+apiVersion: scheduling.k8s.io/v1
+description: Used for system critical pods that must not be moved from their current
+  node.
+kind: PriorityClass
+metadata:
+  name: system-node-critical
+value: 900001000
```
Comment (avishnu) [inline templates/openebs-operator-2.6.0.yaml:5873]: Not sure if the naming of the PriorityClass is correct? Shouldn't it be pre-fixed with "openebs" as in v2.5.0 ?

Comment (Ab-hishek) [reply templates/openebs-operator-2.6.0.yaml:5873]: In the statefulset and ds of CSI in v2.6.0 the name is like this.

**Review by vharsh — APPROVED** [review]
lgtm

File: templates/openebs-operator-2.6.0.yaml [hunk 2]
```diff
@@ -5854,4 +5854,20 @@ spec:
...
-            periodSeconds: 60
\ No newline at end of file
+            periodSeconds: 60
+---
+apiVersion: scheduling.k8s.io/v1
+description: Used for system critical pods that must run in the cluster, but can be
+  moved to another node if necessary.
+kind: PriorityClass
+metadata:
+  name: system-cluster-critical
```
Comment (prateekpandey14) [inline templates/openebs-operator-2.6.0.yaml:5864]: these priorityclass `system-cluster-critical` and `system-node-critical` are created by the k8s by default and should not be modified , can we remove this and add the custom ones, similar to cstor helm charts

**Pull request merged**